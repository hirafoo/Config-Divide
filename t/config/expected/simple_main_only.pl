return +{
    'main2_overwritten_by_sub2_scalar' => 'n1',
    'main2_overwritten_by_sub2_hash' => {
        'n3' => 'n4',
        'n1' => 'n2'
    },
    'main1_overwritten_by_sub1_scalar' => 'n1',
    'main1_only_hash' => {
        'i1' => 'i2',
        'i3' => 'i4'
    },
    'main1_only_scalar' => 'i',
    'main2_only_scalar' => 'i',
    'main2_only_hash' => {
        'i1' => 'i2',
        'i3' => 'i4'
    },
    'main1_overwritten_by_sub1_hash' => {
        'n3' => 'n4',
        'n1' => 'n2'
    }
};
