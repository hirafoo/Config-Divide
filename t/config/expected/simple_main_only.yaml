main1_only_scalar: i
main1_only_hash:
  i1: i2
  i3: i4
main1_overwritten_by_sub1_hash:
  n1: n2
  n3: n4
main1_overwritten_by_sub1_scalar: n1
main2_only_hash:
  i1: i2
  i3: i4
main2_only_scalar: i
main2_overwritten_by_sub2_hash:
  n1: n2
  n3: n4
main2_overwritten_by_sub2_scalar: n1
